src/Cargo.toml

```diff
@@ -16,6 +16,14 @@ members = [
   "tools/rust-installer",
   "tools/cargo",
   "tools/rls",
+  "tools/rls/test_data/borrow_error",
+  "tools/rls/test_data/completion",
+  "tools/rls/test_data/find_all_refs",
+  "tools/rls/test_data/find_all_refs_no_cfg_test",
+  "tools/rls/test_data/goto_def",
+  "tools/rls/test_data/highlight",
+  "tools/rls/test_data/hover",
+  "tools/rls/test_data/rename",
```

Comment:
Would it be possible to instead specify:

```toml
exclude = [
  "tools/rls/test_data",
]
```

?

(this is a relatively new feature of Cargo)